ckanext/recombinant/commands.py

```diff
@@ -362,6 +365,16 @@ def _remove_broken(self, target_datasets):
                         lc.action.package_delete(id=d['id'])
                         break
 
+    def _update(self, target_datasets):
+        """
+        Low-level command to update field value in datasets' datastore tables
+        """
+        lc = LocalCKAN()
+        for dtype in target_datasets:
+            datasets = lc.action.package_search(q="type:%s" % dtype, rows=5000)
+            for d in datasets['results']:
+                for r in d['resources']:
```

Comment:
please add a print statement with the owner-org (and maybe the rows updated) so we can see progress while this command runs.